spec/lib/evss/disability_compensation_form/data_translation_all_claim_spec.rb

```diff
@@ -586,42 +584,6 @@
           'bankName' => 'test'
         }
       end
-
-      context 'when the disability_526_block_banking_info_retrieval Flipper is enabled' do
-        before do
-          allow(Flipper).to receive(:enabled?).with(:disability_526_block_banking_info_retrieval).and_return(true)
-        end
-
-        after do
-          allow(Flipper).to receive(:enabled?).with(:disability_526_block_banking_info_retrieval).and_return(false)
-        end
-
-        it 'logs the submission was made with banking info' do
-          expect_any_instance_of(DisabilityCompensation::Loggers::Monitor)
```

Comment:
If the instances of `DisabilityCompensation::Loggers::Monitor` are only used where removed here, then the corresponding `require` on L7 becomes unused and can safely be deleted